android/versioned-abis/expoview-abi36_0_0/src/main/AndroidManifest.xml

```diff
@@ -108,7 +108,7 @@
         <!-- https://developer.android.com/guide/topics/manifest/provider-element.html -->
         <provider
             android:name="abi36_0_0.expo.modules.imagepicker.ImagePickerFileProvider"
-            android:authorities="abi36_0_0.expo.modules.imagepicker.ImagePickerFileProvider"
+            android:authorities="${applicationId}.ImagePickerFileProvider"
```

Comment:
hmmm, I'm not sure these should be changed? According to https://developer.android.com/guide/topics/manifest/provider-element, the authority is typically
> the name of the ContentProvider subclass that implements the provider

which is this case is the `abi36_0_0.expo.....` class. But perhaps it doesn't matter? 🤷‍♂